### DISCUSSION TITLE
Havera controle de acesso (usuario/senha) para organizadores e administradores?

### DISCUSSION THREAD

**Discussion by jenrodrigues** [andrebaltieri/AgendaEventos#8] [Q&A] [answered]
Me parece que o product backlog atual nao cobre essa feature.

**Comment by renebentes** [comment]
Eu entendo como positiva essa feature. Sem ela apenas adms incluiriam os eventos, eu vejo que isso vai impactar no processo.

**Comment by LuanFreitasRibeiro** [comment, marked as answer]
Como o @renebentes disso, é válido. Podemos controlar isso com Roles e controlar quem tem permição para determinados endpoints...

**Reply by LeonardoPedrosa** [reply]
Penso da mesma forma @LuanFreitasRibeiro. Acredito que essa seja a melhor solução.

**Comment by BrewertonSantos** [comment]
Fazendo ligação com a #10 , neste momento não é prioridade mas é muito aberto não tratar isto. Não tenho detalhes na mente sobre o momento em que conversamos sobre isto.
